daffodil-runtime2/src/main/resources/examples/ex_int32.c

```diff
@@ -149,7 +152,7 @@ c2_parseSelf(c2 *instance, const PState *pstate)
     const char *error_msg = NULL;
     if (!error_msg)
     {
-        char   buffer[4];
+        char   buffer[sizeof(uint32_t)];
```

Comment:
Why does this use uint32_t instead of int32_t ? For consistency's sake, we should use int32_t I think.